pybullet-helpers/src/pybullet_helpers/geometry.py

```diff
@@ -35,7 +35,9 @@ class Pose(NamedTuple):
     @classmethod
     def from_rpy(cls, translation: Pose3D, rpy: RollPitchYaw) -> Pose:
         """Create a Pose from translation and Euler roll-pitch-yaw angles."""
-        return cls(translation, quaternion_from_euler(*rpy))
+        # quaternion_from_euler returns an ndarray; convert to tuple for the
```

Comment:
can you create a new helper function to avoid multiple uses of casts. it can be like this:

```python
from pybullet_utils.transformations import quaternion_from_euler as putil_quat_from_euler

def quaternion_from_euler(rpy: RollPitchYaw) -> Quaternion:
    """Convert a euler angle into a quaternion."""
    qx, qy, qz, qw = putil_quat_from_euler(*rpy)
    return (qx, qy, qx, qw)
```

use here and below, and remove all instances of `cast`. thanks!